ReadMe.md

```diff
@@ -0,0 +1,4 @@
+# Steps:-
+1. clone the repo
```

Comment:
Add more info, like what does the App do and add more information to make the user understand